test/Pattern/ClassCacheTest.php

```diff
@@ -99,20 +99,20 @@ protected function _testCall($method, array $args)
         $return = call_user_func_array([$this->_pattern, $method], $args);
```

Comment:
```suggestion
        $return = ($this->_pattern)->$method(...$args);
```

Basically, I'd suggest looking for all `call_user_func_array` usage and updating to use the splat operator.